### PR TITLE
phase4and5

### PR DESCRIPTION
**Pull request by JuliaTDunc**

### CODE REVIEW

File: backend/db/migrations/20240314022512-create-user.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
+      firstName: {
```
Comment (AlanDeleon88) [inline backend/db/migrations/20240314022512-create-user.js:15]: You should have allow null false for firstName and lastName columns. Also it is a good idea to create a new migration to add the columns, especially if the app is deployed. Migrations can act as a type of version control for your database.

File: backend/db/models/user.js [hunk 2]
```diff
@@ -26,6 +26,12 @@ module.exports = (sequelize, DataTypes) => {
         }
       }
     },
+    firstName: {
```
Comment (AlanDeleon88) [inline backend/db/models/user.js:29]: Should add model validations to firstName and lastName. You should check if a name is too long or an empty string. Also it's a good idea to add allowNull to be false.

**Review by AlanDeleon88 — CHANGES_REQUESTED** [review]
Go over the comments for your new columns in the User table/model. Should be good to go after those changes are made.

File: backend/db/migrations/20240314022512-create-user.js (hunk 1 above)
Comment (JuliaTDunc) [reply backend/db/migrations/20240314022512-create-user.js]: Just finished the new migration file. It should be visible to you now. I removed the firstName and lastName columns that I added to the original migration file as well, if thats alright

**Pull request merged**